src/Question.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Question -represents a question for the plugin CLI
+ */
+class Question {
+
+	/**
+	 * constructor - Question
+	 *
+	 * @param  {string} name              name of option
```

Comment:
I wonder if we should use `string` or `String`?

Reply:
I think we should write `string` when we talk about primitives and `String` for objects.